src/go-camo/camo/proxy.go

```diff
@@ -21,6 +24,15 @@ import (
 	"github.com/cactus/mlog"
 )
 
+var acceptedContentTypes = []string{
+	"application/font-woff",
+	"application/vnd.ms-fontobject",
+	"application/x-font",
+	"image/",
+	"text/css",
+}
+var defaultAcceptHeaders = "image/*, text/*, application/*"
```

Comment:
This might be a bit pedantic (since you check later for the relevant content types anyway), but could you get away with reusing the string list immediately above?

Reply:
Yes, I have considered that. The problem is, `acceptedContentTypes` technically allows for some level of _fuzziness_ (e.g. `image/`), whereas `defaultAcceptHeaders` needs to define a valid content type that may or may not contain wildcards. Additionally, the order matters as well. For example, if `application/*` comes prior to `text/*`, the format of the response may be JSON instead of text. Perhaps the solution is just to reconcile the string syntax. That is, use wildcards rather than just assume that it is a prefix, and it is OK. For now, a bit of duplication just seems simpler.